src/stream_management/mod_stream_management_stale_h.erl

```diff
@@ -0,0 +1,136 @@
+-module(mod_stream_management_stale_h).
+-behaviour(gen_server).
+
+-include("mongoose.hrl").
+-include_lib("stdlib/include/ms_transform.hrl").
+
+-record(state,
```

Comment:
Try to avoid introducing new state records.
Basically, it's good to do prefixes to state records i.e. `#smsh_state{}`.

Reasoning: sometimes we need to load records into shell, duplicates are confusing.